$NetBSD: patch-aa,v 1.3 2006/12/13 02:00:29 obache Exp $

NetBSD 3.0 needs -liconv, too.

Install perl modules into the "vendor" directories.

--- Makefile.orig	2006-11-23 02:38:07.000000000 +0000
+++ Makefile
@@ -393,9 +393,7 @@ ifeq ($(uname_S),OpenBSD)
 	BASIC_LDFLAGS += -L/usr/local/lib
 endif
 ifeq ($(uname_S),NetBSD)
-	ifeq ($(shell expr "$(uname_R)" : '[01]\.'),2)
-		NEEDS_LIBICONV = YesPlease
-	endif
+	NEEDS_LIBICONV = YesPlease
 	BASIC_CFLAGS += -I/usr/pkg/include
 	BASIC_LDFLAGS += -L/usr/pkg/lib
 	ALL_LDFLAGS += -Wl,-rpath,/usr/pkg/lib
@@ -769,7 +767,7 @@ $(XDIFF_LIB): $(XDIFF_OBJS)
 
 perl/Makefile: perl/Git.pm perl/Makefile.PL GIT-CFLAGS
 	(cd perl && $(PERL_PATH) Makefile.PL \
-		PREFIX='$(prefix_SQ)')
+		INSTALLDIRS=vendor PREFIX='$(prefix_SQ)')
 
 doc:
 	$(MAKE) -C Documentation all
